packages/components/src/slot-fill/bubbles-virtually/fill.tsx

```diff
@@ -45,9 +27,10 @@ export default function Fill( { name, children }: FillComponentProps ) {
 		// We register fills so we can keep track of their existence.
 		// Some Slot implementations need to know if there're already fills
 		// registered so they can choose to render themselves or not.
-		registry.registerFill( name, ref );
+		const refValue = ref.current;
+		registry.registerFill( name, refValue );
 		return () => {
-			registry.unregisterFill( name, ref );
+			registry.unregisterFill( name, refValue );
```

Comment:
Instead of registering the `ref` object I'm registering directly the `Rerenderable` value. Which is also constant.